dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/client/jose4j/util/Jose4jUtil.java

```diff
@@ -183,6 +188,18 @@ public ProviderAuthenticationResult createResultWithJose4J(String responseState,
         return oidcResult;
     }
 
+    private void addHttpSessionToCache(OidcClientRequest oidcClientRequest, JwtClaims jwtClaims, ConvergedClientConfig clientConfig) throws MalformedClaimException {
+        OidcClientConfig oidcClientConfig = clientConfig.getOidcClientConfig();
+        if (oidcClientConfig.isBackchannelLogoutSupported()) {
+            String sub = jwtClaims.getSubject();
+            String sid = jwtClaims.getClaimValue("sid", String.class);
+            String httpSessionId = oidcClientRequest.getRequest().getSession().getId();
```

Comment:
Similar comment here as before. Not sure if we need to do something like this:
```suggestion
            HttpSession session = oidcClientRequest.getRequest().getSession(false);
            String httpSessionId = (session == null) ? null : session.getId();
```

Reply:
ditto